public/test/setupTests.ts

```diff
@@ -1,12 +1,16 @@
 import '@testing-library/jest-dom';
 import { configure } from '@testing-library/react';
+import dotenv from 'dotenv';
 import i18next from 'i18next';
 import failOnConsole from 'jest-fail-on-console';
 import { initReactI18next } from 'react-i18next';
 
+dotenv.config();
+dotenv.config({ path: `.env.local`, override: true });
+
 import { matchers } from './matchers';
 
-if (process.env.CI) {
+if (process.env.GRAFANA_TESTS_FAIL_ON_CONSOLE || process.env.CI) {
```

Comment:
i wonder whether we should just remove the `process.env.CI` check and default this to enabled in the `.env` file? 🤔

Reply:
Hmm yes good point, we could do that - I saw that @joshhunt had enabled it in CI only in https://github.com/grafana/grafana/pull/67313, but we could perhaps "undo" that and allow people to opt back out of it if they wanted to?

Reply:
I think `process.env.CI` is like the common pattern to do things like this... It might be neat if the `GRAFANA_TESTS_FAIL_ON_CONSOLE` default value was set from `CI`, but I don't want to overcomplicate things too much...

Reply:
>i wonder whether we should just remove the process.env.CI check and default this to enabled in the .env file?

I don't think I agree with this. I don't think there should be a `.env` file in the repo as the source of truth. Also which will take precedence right now if we set `CI=false` in the `.env` and pass it as true in a drone CI step?